falco/values.yaml

```diff
@@ -1,11 +1,149 @@
 # Default values for Falco.
 
+###############################
+# General deployment settings #
+###############################
+
 image:
-  registry: docker.io
-  repository: falcosecurity/falco
-  tag: 0.32.0
   pullPolicy: IfNotPresent
-  pullSecrets: []
+  registry: docker.io
+  repository: falcosecurity/falco-no-driver
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+rbac:
+  # Create and use rbac resources
+  create: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# Add additional pod annotations
+podAnnotations: {}
+
+# Add additional pod labels
+podLabels: {}
+
+# Set pod priorityClassName
+podPriorityClassName:
+
+# Set securityContext for the pods
+podSecurityContext: {}
+
+# Set securityContext for the DaemonSet/Deployment containers
+# This block works in combination with the `leastPrivileged` block above.
+securityContext: {}
+
+scc:
+  # true here enabled creation of Security Context Constraints in Openshift
+  create: true
+
+resources:
+  # Although resources needed are subjective on the actual workload we provide
+  # a sane defaults ones. If you have more questions or concerns, please refer
+  # to #falco slack channel for more info about it
+  requests:
+    cpu: 100m
+    memory: 512Mi
+  limits:
+    cpu: 1000m
+    memory: 1024Mi
+
+nodeSelector: {}
+
+affinity: {}
+
+# Allow Falco to run on Kubernetes 1.6 masters.
+tolerations:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
```

Comment:
I would remove the toleration from here since it is always needed in order to schedule Falco on controller-plane nodes. And if we really need different `toleration` based on k8s version then we could use `Capabilities.KubeVersion`.